py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -33,7 +33,7 @@ def create_event(self) -> dict:
             'files': self.filenames,
         }
 
-def batch_files(files: Iterable[(str, int)], threshold: int) -> list[Batch]:
+def batch_files(files: Iterable[(str, int)], threshold: int) -> Iterable[Batch]:
```

Comment:
made this a generator function. it felt more right when i did a large test run today.

this way we can trigger jobs as we know about them rather than triggering a bunch at the end.